config/locales/sv.yml

```diff
@@ -79,6 +82,14 @@ sv:
         email: &email E-post
         password: *password
         created: &created Skapad
+        current_sign_in_at: &user_current_signed_in_time Inloggad pä
+        current_sign_in_ip: &user_current_ip Inloggad pä IP
+        last_login: &user_last_login Senast inloggad
+        last_login_ip: &user_last_login_ip Senast inloggad IP
+        sign_in_count: &sign_in_count Inloggningar
+        created: *created
+        reset_password_sent_at: &user_reset_password_sent_at Återställning av lösenord information sändes
```

Comment:
No biggie, but while we're at it: 
Information angående återställning av lösenord sändes